Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:bullseye-slim
+FROM public.ecr.aws/docker/library/debian:bullseye-slim
```

Comment:
I don't understand why this is needed? The default `debian:bullseye-slim` should work fine?

Reply:
@MyDapitt Yeah, I don't really see why this needs to be changed, can you please revert this line.